src/apps/bm_soft_module/user_code/user_code.cpp

```diff
@@ -41,49 +46,59 @@ static bool BmSoftStartAndValidate(void);
+    printf("TSYS serial number: %" PRIu32 "\n", serial_number);
+  } else {
+    printf("No TSYS serial number found\n");
+    bm_fprintf(0, soft_log, "No TSYS serial number found\n");
+    bm_printf(0, "No TSYS serial number found");
+  }
+
   int32_t calTempC = 0;
-  int32_t calTempMilliC = 0;
-  if (sysConfigurationPartition->getConfig("calTempC", strlen("calTempC"), calTempC)) {
-    printf("calTempC: %" PRId32 "\n", calTempC);
+  if (sysConfigurationPartition->getConfig(soft_cfg_cal_temp_c, strlen(soft_cfg_cal_temp_c),
+                                           calTempC)) {
+    printf("Calibration temperature: %" PRId32 "\n", calTempC);
   } else {
     printf("No calibration temperature found\n");
     bm_fprintf(0, soft_log, "No calibration temperature found\n");
     bm_printf(0, "No calibration temperature found");
   }
-  if (sysConfigurationPartition->getConfig("tsysId", strlen("tsysId"), serial_number)) {
-    printf("SOFT Serial Number: %" PRIu32 "\n", serial_number);
-  } else {
-    printf("No calibration serial number found\n");
-    bm_fprintf(0, soft_log, "No calibration serial number found\n");
-    bm_printf(0, "No calibration serial number found");
-  }
-  if (sysConfigurationPartition->getConfig("calTimeEpoch", strlen("calTimeEpoch"),
-                                           cal_time_epoch)) {
-    printf("Cal time: %" PRIu32 "\n", cal_time_epoch);
+
+  if (sysConfigurationPartition->getConfig(soft_cfg_cal_time_epoch,
+                                           strlen(soft_cfg_cal_time_epoch), cal_time_epoch)) {
+    printf("Calibration time: %" PRIu32 "\n", cal_time_epoch);
   } else {
     printf("No calibration time found\n");
     bm_fprintf(0, soft_log, "No calibration time found\n");
     bm_printf(0, "No calibration time found");
   }
-  if (sysConfigurationPartition->getConfig("calTempMilliC", strlen("calTempMilliC"),
-                                           calTempMilliC)) {
-    printf("calTempMilliC: %" PRId32 "\n", calTempMilliC);
+
+  int32_t calOffsetMilliC = 0;
+  if (sysConfigurationPartition->getConfig(
+          soft_cfg_cal_offset_milli_c, strlen(soft_cfg_cal_offset_milli_c), calOffsetMilliC)) {
+    printf("Calibration offset (milliDegC): %" PRId32 "\n", calOffsetMilliC);
   } else {
-    printf("No calibration temperature (milliDegC) found\n");
-    bm_fprintf(0, soft_log, "No calibration temperature (milliDegC) found\n");
-    bm_printf(0, "No calibration temperature (milliDegC) found");
+    printf("No calibration offset (milliDegC) found\n");
+    bm_fprintf(0, soft_log, "No calibration offset (milliDegC) found\n");
+    bm_printf(0, "No calibration offset (milliDegC) found");
   }
-  calibrationOffsetDegC = (calTempC + (calTempMilliC / 1000.0f));
-  if (userConfigurationPartition->getConfig("softReadingPeriodMs",
-                                            strlen("softReadingPeriodMs"), soft_delay_ms)) {
+  calibrationOffsetDegC = static_cast<float>(calOffsetMilliC) * 1e-3f;
```

Comment:
Doesn't this need to incorporate the `calTempC` term?

Reply:
From my reading of the spec, no. It seems like `calTempC` is just informational. @evanShap could you please give us a reality check on what precisely gets programmed into the calibration offset config and how it should be used?

Reply:
The spec has this note:
> Note - calibration offset is stored such that: true_temp = reading_temp + calibration_offset.

Reply:
Got it - if this is true we should update Mike with this info.

Reply:
Just saw the Notes section of the config! Approving!